pkgs/tools/networking/litmus/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, openssl }:
+
+stdenv.mkDerivation rec {
+  version = "0.13";
+  name = "litmus-${version}";
+
+  src = fetchurl {
+    url = "http://webdav.org/neon/litmus/${name}.tar.gz";
+    sha256 = "09d615958121706444db67e09c40df5f753ccf1fa14846fdeb439298aa9ac3ff";
+  };
+
+  buildInputs = [ openssl ];
+  configureFlags = [ "--with-ssl" ];
+  installFlags = [ "PREFIX=$(out)" ];
```

Comment:
Are you sure this is needed?

Reply:
Yes. Without "openssl"/"--with-ssl", litmus would only be able to communicate over http, not https. And https/SSL is important.

inputFlags probably may be omitted, I think.

Reply:
(minor correction: do you mean `installFlags` can be dropped, not `inputFlags`?)

Reply:
Yes, installFlags may be dropped, I think.

Reply:
Yes, I was refering to line 14, ``installFlags``.